### PR TITLE
SQL Server Docs

### PR DESCRIPTION
**Pull request by djova**
### What does this PR do?

Add new SQL Server documentation for Database Monitoring that covers the other managed SQL Server instance types we support in Azure & AWS. 

### Preview
* [selfhosted](https://docs-staging.datadoghq.com/djova/dbm-sqlserver-initial-pages/database_monitoring/setup_sql_server/selfhosted/)
* [azure](https://docs-staging.datadoghq.com/djova/dbm-sqlserver-initial-pages/database_monitoring/setup_sql_server/azure/)
* [rds](https://docs-staging.datadoghq.com/djova/dbm-sqlserver-initial-pages/database_monitoring/setup_sql_server/rds/)

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/database_monitoring/setup_sql_server/rds.md
Comment (jmeunier28) [inline content/en/database_monitoring/setup_sql_server/rds.md:43]: I'm fairly confident this command fails on RDS, and I think to fix some of the metrics issues I saw I had to create the DD user in every single DB on the host 

there are some details about this in the description of this [jira ticket](https://datadoghq.atlassian.net/browse/DBM-997)

**Comment by apigirl** [comment]
editorial review card https://datadoghq.atlassian.net/browse/DOCS-3315

File: content/en/database_monitoring/setup_sql_server/rds.md
Comment (djova) [reply content/en/database_monitoring/setup_sql_server/rds.md]: Good catch! Fixed https://github.com/DataDog/documentation/pull/13478/commits/a74987759ec4f72134473851135c508ebe903a83

File: layouts/shortcodes/dbm-sqlserver-agent-setup-kubernetes.md
Comment (cswatt) [inline layouts/shortcodes/dbm-sqlserver-agent-setup-kubernetes.md:3]:
```suggestion
If cluster checks are not already enabled in your Kubernetes cluster, follow the instructions to [enable cluster checks][2]. You can configure the Cluster Agent either with static files mounted in the Cluster Agent container, or by using Kubernetes service annotations:
```

File: layouts/shortcodes/dbm-sqlserver-agent-setup-kubernetes.md
Comment (cswatt) [inline layouts/shortcodes/dbm-sqlserver-agent-setup-kubernetes.md:26]:
```suggestion
  datadog/datadog"
```

seems like a `"` was missing here

File: layouts/shortcodes/dbm-sqlserver-agent-setup-kubernetes.md
Comment (cswatt) [inline layouts/shortcodes/dbm-sqlserver-agent-setup-kubernetes.md:79]:
```suggestion
The Cluster Agent automatically registers this configuration and begins running the SQL Server check.
```

File: layouts/shortcodes/dbm-sqlserver-agent-setup-linux.md
Comment (cswatt) [inline layouts/shortcodes/dbm-sqlserver-agent-setup-linux.md:3]:
```suggestion
On Linux, the Datadog Agent additionally requires an ODBC SQL Server driver to be installed—for example, the [Microsoft ODBC driver][2]. Once an ODBC SQL Server is installed, copy the `odbc.ini` and `odbcinst.ini` files into the `/opt/datadog-agent/embedded/etc` folder.
```

File: layouts/shortcodes/dbm-sqlserver-agent-setup-linux.md
Comment (cswatt) [inline layouts/shortcodes/dbm-sqlserver-agent-setup-linux.md:7]:
```suggestion
Create the SQL Server Agent conf file `/etc/datadog-agent/conf.d/sqlserver.d/conf.yaml`. See the [sample conf file][3] for all available configuration options.
```

File: layouts/shortcodes/dbm-sqlserver-agent-setup-linux.md
Comment (cswatt) [inline layouts/shortcodes/dbm-sqlserver-agent-setup-linux.md:25]:
```suggestion
Once all Agent configuration is complete, [Restart the Datadog Agent][5].
```

File: layouts/shortcodes/dbm-sqlserver-agent-setup-linux.md
Comment (cswatt) [inline layouts/shortcodes/dbm-sqlserver-agent-setup-linux.md:29]:
```suggestion
[Run the Agent's status subcommand][6] and look for `sqlserver` under the **Checks** section. Navigate to the [Databases][7] page in Datadog to get started.
```

File: layouts/shortcodes/dbm-sqlserver-agent-setup-windows.md
Comment (cswatt) [inline layouts/shortcodes/dbm-sqlserver-agent-setup-windows.md:5]:
```suggestion
* **ODBC**: set `connector: odbc`, `driver: {ODBC Driver 17 for SQL Server}`, and ensure the [Microsoft ODBC Driver][3] is installed on the host where the Agent is running.
```

File: layouts/shortcodes/dbm-sqlserver-agent-setup-windows.md
Comment (cswatt) [inline layouts/shortcodes/dbm-sqlserver-agent-setup-windows.md:7]:
```suggestion
Create the SQL Server Agent conf file `C:\ProgramData\Datadog\conf.d\sqlserver.d\conf.yaml`. See the [sample conf file][4] for all available configuration options.
```

File: layouts/shortcodes/dbm-sqlserver-agent-setup-windows.md
Comment (cswatt) [inline layouts/shortcodes/dbm-sqlserver-agent-setup-windows.md:27]:
```suggestion
Once all Agent configuration is complete, [Restart the Datadog Agent][6].
```

File: layouts/shortcodes/dbm-sqlserver-agent-setup-windows.md
Comment (cswatt) [inline layouts/shortcodes/dbm-sqlserver-agent-setup-windows.md:31]:
```suggestion
[Run the Agent's status subcommand][7] and look for `sqlserver` under the **Checks** section. Navigate to the [Databases][8] page in Datadog to get started.
```

**Review by cswatt — CHANGES_REQUESTED** [review]

**Comment by jtappa** [comment]
Waiting on release/go ahead from DBM before merging

**Pull request merged**
